### PR TITLE
Styling Bina Box

### PR DESCRIPTION
**Pull request by EvgeniiGulev**

### CODE REVIEW

**Review by arkhipenkav — COMMENTED** [review]
Hard work was done here. It is impressive. But there is still a space for improvements.

**What I liked:**
- you connected correct styles for all the pages in a proper order
- you added fonts correctly
- you practiced with different ways of inserting images 
- you practiced with very difficult properties, css variables and etc

**What must be improved.**
Introduction in CSS
- avoid to style tags directly and use * (global selector)
- add proper common font styles and background for the body
- remove repeated inherited styles. For example, you reapply font-family 15 times through all the documents. If you will add Poppins for the body, you do not need to add it anywhere else, it is inherited. color, font-weight, etc work the same. 
-  remove default styles. e.g. `text-align: left`

**Images on the Web**
- read more about a difference between vector and rastr graphics. Check all the images and export it properly. Add more efficient formats
- insert svg in html in a proper way

**Forms**
- check form attributes to make forms valid
- `input type="checkbox"` has a required attribute, you must add it

What can be improved:
- use svg sprites
- optimize images
- try to add in filters `fieldset` and `legend` to group checkboxes
- rework slider with cards that repeated on every page, make the same structure and add similar classes. Then remove repeated styles. It works well with banner :)
- structure in styles.css is very difficult to read. It seems like you used some tool as a helper. If yes, I do not recommend you to do that on this step, but it is up to you :) It is not forbidden. But it is better for a learning process to understand every property that you have added and why it is needed. If no, so just do not duplicate code and try to do it more clear with some comments :)

File: css/style.css [hunk 1]
```diff
@@ -1 +1,2142 @@
...
+.footer-navbar,
+.navbar,
+.nft-cards-list,
+.phase-line,
+.road-map-details,
+.team-inner-container ul {
+  list-style-type: none;
+}
+:root {
+  --back-black-1: #0d0d11;
```
[the remark this answers is not in the document]
Comment (EvgeniiGulev) [reply css/style.css:44]: <3

File: css/style.css [hunk 2]
```diff
@@ -1 +1,2142 @@
...
+.main-heading,
+.main-text,
+.nav-logo {
+  font-weight: inherit;
+}
+.hidden-heading {
+  display: none;
+}
+.color-highlight {
+  background: #fff0;
```
[the remark this answers is not in the document]
Comment (EvgeniiGulev) [reply css/style.css:102]: glow container is the corner glow style its not a highlight

File: css/style.css [hunk 3]
```diff
@@ -1 +1,2142 @@
...
+  font-family: Poppins, Arial, sans-serif;
+}
+.about-heading,
+.about-text,
+.connect-wallet-btn,
+.header-btn,
+.main-heading,
+.main-text,
+.nav-logo {
+  font-weight: inherit;
```
[the remark this answers is not in the document]
Comment (EvgeniiGulev) [reply css/style.css:96]: Not sure why but for some reason the font weight is different for some texts and applying inherit fixes it.

File: css/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (EvgeniiGulev) [reply css/style.css:102]: I used a mark for color-highlight but thinking about it I will change it to span

**Review by arkhipenkav — COMMENTED** [review]
Good job! You have corrected almost all the comments and the code looks much better now.

What I've liked:
- css file is shorter for 600 rows, so you checked repeated styles! I'm sure it was not an easy job, very good.
- correct images formats, usage of sprite
- you have started to use aria-labels, that is a good practice. But don't overdo it :)
- forms are vavid now, and it's great that inputs have a basic validation
- no styled tags only classes 🥇 

What must be improved:
- check required attribute for checkboxes
- have a look one more time to repeated styles. Do not forget about inheritance!

The rest of comments are suggestions. Good luck!

**Comment by EvgeniiGulev** [comment]
I believe I fixed everything that you commented on :)

**Review by arkhipenkav — APPROVED** [review]
Approved :) You can merge it now

**Pull request merged**